scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -382,6 +481,24 @@ check_agreement()
     done
 }
 
+check_specific_generation()
+{
+    echo "Checking processor generation..."
+    specific_generation=$(cat /proc/cpu-info | grep -m1 'model name' | grep -E "i[357]-1[01][0-9]{2,4}N?G[147R]E?")
+    if [[ ! -z $specific_generation && $INSTALL_DRIVER_VERSION != '20.35.17767' ]]; then
+        echo "$(basename "$0"): Detected 10th generation Intel® Core™ processor (formerly Ice Lake) or 11th generation Intel® Core™ processor (formerly Tiger Lake)."
+        echo "Driver version 20.35.17767 is going to be installed to fully utilize hardware features and performance."
+        while true; do
+            read -p "You are still able to use the older version 19.41.14441. Use the older driver? (y/n) [n] " yn
```

Comment:
Maybe this question should be omitted with `-y` option too?

Reply:
Different behaviour - it can be ommited with -d option

Reply:
When running with `-y`, users assume no questions will be asked, i.e. fully automatic behavior (detection and installation). While `-d` option affects only detection stage.